specification/network/resource-manager/Microsoft.Network/stable/2021-02-01/applicationGateway.json

```diff
@@ -1509,6 +1509,16 @@
           "type": "string",
           "description": "Certificate public data."
         },
+        "validatedCertData": {
+          "readOnly": true,
```

Comment:
It seems you are putting secret data into the GET response, is this a valid behavior?  Will service return the actual value of cert data?

Reply:
Hi @lirenhe , thanks for the review, "validatedCertData" is same as "data" property which is public data.